docs/guides/cloud.rst

```diff
@@ -77,32 +77,59 @@ If you'd prefer, you can also manage your account via `the EdgeDB Cloud
 web-based GUI <https://cloud.edgedb.com/>`_.
 
 The first time you access the web UI, you will be prompted to log in. Once you
-log in with your GitHub account, you'll see your instance list.
+log in with your account, you'll be on the Instances tab of the front page
+which shows your instance list. The other two tabs allow you to manage your
+Org Settings, and Billing.
+
+Instances
+---------
 
 If this is your first time accessing EdgeDB Cloud, this list will be empty. To
-create an instance, click "Create new instance." This will pop up a modal
+create an instance, click "Create new instance". This will pop up a modal
 allowing you to name your instance and specify the version of EdgeDB and the
 region for the instance.
 
 Once the instance has been created, you'll see the instance dashboard which
-allows you to monitor your instance, navigate to the database management page,
-and create secret keys.
+allows you to monitor your instance, navigate to the management page for its
+database(s), and create secret keys.
```

Comment:
I think this looks nicer and is still OK even if the instance has only one database.
```suggestion
databases, and create secret keys.
```